anda/lib/process-cpp/process-cpp.spec

```diff
@@ -0,0 +1,84 @@
+%global forgeurl https://gitlab.com/ubports/development/core/lib-cpp/process-cpp
+%global commit ee6d99a3278343f5fdcec7ed3dad38763e257310
```

Comment:
I think same issue here unfortunately, wrong commit

Reply:
Meaning what? There are some changes/commits not in `3.0.1`.

Reply:
Hmm, maybe just name the commit `3.0.2^%commit`